plugins/inputs/icinga2/README.md

```diff
@@ -25,8 +25,10 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
   ## Required Icinga2 server address
   # server = "https://localhost:5665"
   
-  ## Required Icinga2 object type ("services" or "hosts")
-  # object_type = "services"
+  ## Icinga2 object type ("services" or "hosts") from /v1/objects endpoint
+  # objects = ["services", "hosts"]
```

Comment:
Sorry, I haven't noticed the deprecation framework. I've added the deprecation notice. Is it ok, to leave it without removal version? Or should I set it to a version in a far away future? e.g. `2.0`?

Reply:
I would set it to `2.0` for now.

Reply:
In test mode it now prints this message, when `object_type` is specified
```
2023-01-18T14:14:53Z W! DeprecationWarning: Option "object_type" of plugin "inputs.icinga2" deprecated since version 1.26.0 and will be removed in 2.0.0: use 'objects' instead
2023-01-18T14:14:53Z W! DeprecationWarning: Option "object_type" of plugin "inputs.icinga2" deprecated since version 1.26.0 and will be removed in 2.0.0: use 'objects' instead
2023-01-18T14:14:53Z I! Starting Telegraf 1.26.0-89845d68
```
Now sure why the message is displayed twice, but otherwise it looks good.